src/org/scada_lts/service/model/MultiChangeHistoryDTO.java

```diff
@@ -0,0 +1,113 @@
+package org.scada_lts.service.model;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * @author  grzegorz.bylica@abilit.eu on 15.10.2019
+ */
+public class MultiChangeHistoryDTO {
+
+
+    public transient final static int NEW_ID = -1;
+
+    transient private int id;
+    private String userName;
+    private int userId;
+    private long unixTime;
+    private String interpretedState;
+    private Set<ValuesMultiChangesHistoryDTO> values;
```

Comment:
Is it possible to change names "values" to more understandable . It means that valuesMultiChangeHistoryDTO. DTO tells that this is DTO.

Reply:
I'd rather not change the name